src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -23,16 +22,14 @@
 using Microsoft.VisualStudio.R.Package.Repl;
 
 namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
-    public sealed class RInteractiveWorkflow : IRInteractiveWorkflow {
+    public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
```

Comment:
`GetOrCreateVisualComponentAsync` goes to the `IInteractiveWindowVisualProvider`. So it becomes:
```cs
public interface IInteractiveWindowVisualProvider {
    IInteractiveWindowVisual GetAsync(IRInteractiveWorkflow workflow)
    IInteractiveWindowVisual Current { get; }
    event EventHandler<ActiveWindowChangedEventArgs> CurrentChanged;
}
```

In CM and PM, there will be only `Get` method.